exonum/src/node/mod.rs

```diff
@@ -342,6 +343,7 @@ pub struct NodeConfig {
     /// Memory pool configuration.
     pub mempool: MemoryPoolConfig,
     /// Additional config, usable for services.
+    #[serde(default)]
     pub services_configs: BTreeMap<String, Value>,
     /// Optional database configuration.
     pub database: Option<DbOptions>,
```

Comment:
What about adding attribute for this field as well? (And then we can remove `Option`? :thinking: )